app/models/lookup_key.rb

```diff
@@ -214,7 +216,7 @@ def array2path(array)
   def validate_and_cast_default_value
     return true if default_value.nil? || contains_erb?(default_value)
     begin
-      self.default_value = cast_validate_value self.default_value
```

Comment:
`cast_validate_value` and all the other cast methods are still in this file, are they used?